src/meta/src/backup_restore/utils.rs

```diff
@@ -0,0 +1,34 @@
+// Copyright 2022 Singularity Data
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+use std::hash::Hasher;
+
+use crate::backup_restore::error::{BackupError, BackupResult};
+
+pub fn xxhash64_checksum(data: &[u8]) -> u64 {
```

Comment:
Same with `sstable/utils.rs`, can we unify it ?

Reply:
Will deduplicate in another PR that implements restoring.